src/v2/view-builder/internals/BaseForm.js

```diff
@@ -128,9 +129,25 @@ export default Form.extend({
     // render messages as text
     const messagesObjs = this.options.appState.get('messages');
     if (messagesObjs?.value.length) {
-      const content = messagesObjs.value.map((messagesObj) => {
-        return messagesObj.message;
-      });
+
+      const content = messagesObjs.value
+        .filter(messagesObj => {
+          const isWarningMessage = messagesObj?.class === MESSAGE_CLASS.WARN;
```

Comment:
if there are multiple warnings is it going to show them all ? I think UX would like to just show the first warning widget receives.

Reply:
This is a good point, I didn't account for this. So we just want to display the first warning we come across correct?